app/controllers/comments.controller.js

```
@@ -30,4 +30,23 @@ exports.postComments = (req, res, next) => {
 })  .catch((err) => {
          next(err)
 })
+}
+
+exports.deleteComment = (req, res, next) => {
+    const {comment_id} = req.params
+    
+    if(isNaN(comment_id)){
+        return next({status: 400, msg: "Bad request"})
+    }
```

PSQL should throw an error if the comment_id is not a number 👍